src/util/boost_asio.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
```

Comment:
This whole file is basically unnecessary. If you add post_periodic function to the the threadpool and send threadpool object into the taraxa capability by reference, this can be deleted